### ISSUE TITLE
Initial calendar is incorrect when minDate and defaultActiveStartDate both used

### ISSUE THREAD

**Issue by lloydstubber** [wojtekmaj/react-calendar#356]
Hi there,

We're using your calendar in our app and have just updated to 3.0.0 and are having issues with defaultActiveStartDate.

If you see my sandbox here: https://codesandbox.io/s/old-dew-x87cq?file=/src/App.js

When you first select a date, it will rerender and move the dates within the month view. See gif below of sandbox:

![datepick-issue](https://user-images.githubusercontent.com/18339250/78864513-1893dc00-7a7f-11ea-9f87-a296e5e08aaf.gif)

Note: My colleague has made a more whittled down sandbox here: https://codesandbox.io/s/hardcore-dust-8m8zv?file=/src/App.js

**Comment by ksoldau** [comment]
Hello, colleague here :). I think what's happening is that when the calendar renders, it's showing the calendar days as if they're for the minDate instead of the default active date. Then, when a date is selected, the days shift to their correct placements.

**Comment by ksoldau** [comment]
I went to try and fix the problem myself but realized that it wasn't a problem on the master branch :D. 

Looks like this was fixed by this PR: https://github.com/wojtekmaj/react-calendar/commit/8e610bca7d123982f3f93f513a1dd36cc6785842

And that this issue relates to the already closed [#345 issue](https://github.com/wojtekmaj/react-calendar/issues/345). 

I'm going to close this issue since the fix is in master already. 

@wojtekmaj Thanks for all your work on this! Any plans to release the bugfix? I can't update to 3.0.0 until there's another release.
